files/zh-cn/web/api/node/firstchild/index.html

```diff
@@ -69,10 +69,6 @@ <h3 id=".E6.B3.A8.E6.84.8F" name=".E6.B3.A8.E6.84.8F">备注</h3>
     <p>详情请参见 <a class="new" href="/zh-CN/docs/Whitespace_in_the_DOM" rel="nofollow">DOM 中的空白符</a>
     和<a class="external" href="http://www.w3.org/DOM/faq.html#emptytext" rel="noopener">W3C DOM 3 FAQ: 为什么一些文本节点是空的</a>.</p>
 
-<h3 id="Specification" name="Specification">规范</h3>
+<h3 id="规范">规范</h3>
 
-<p><a class="external" href="http://www.w3.org/TR/REC-DOM-Level-1/level-one-core.html#attribute-firstChild">DOM Level 1 Core: firstChild</a></p>
-
-<p><a class="external" href="http://www.w3.org/TR/DOM-Level-2-Core/core.html#ID-169727388">DOM Level 2 Core: firstChild</a></p>
-
-<p>{{ languages( { "fr": "fr/DOM/element.firstChild", "ja": "ja/DOM/element.firstChild", "pl": "pl/DOM/element.firstChild", "en": "en/DOM/Node.firstChild" } ) }}</p>
```

Comment:
here

Reply:
reverted in 779495d822eb7903e1f179f5b19520efd10c2d63 and d5298880fe186a1b01126b841b8c3c04ecc66999